src/seedu/addressbook/AddressBook.java

```diff
@@ -99,10 +99,10 @@
     private static final String COMMAND_ADD_EXAMPLE = COMMAND_ADD_WORD + " John Doe p/98765432 e/johnd@gmail.com";
 
     private static final String COMMAND_FIND_WORD = "find";
-    private static final String COMMAND_FIND_DESC = "Finds all persons whose names contain any of the specified "
-                                        + "keywords (case-sensitive) and displays them as a list with index numbers.";
+    private static final String COMMAND_FIND_DESC = "Finds all persons whose names start with or contain any of the specified "
```

Comment:
is this change necessary? `contain any of the specified` would have include `start with`